zaga-web/src/app/components/field/field.component.html

```diff
@@ -1,54 +1,172 @@
-<div style="padding-top: 10px"></div>
-<h3 class="mt-2 mb-2">Pregled parcela</h3>
-<div style="padding-top: 20px"></div>
-<div class="containerFields">
-  <div class="rowCard">
-    <div class="columnCard" *ngFor="let field of fields; let j = index">
-      <div class="card mb-3" style="width: 22rem; margin-left: 5px">
-        <div class="card-body">
-          <div class="row">
-            <div
-              class="col-md-auto"
-              style="width: fit-content; padding-top: 12px"
-            >
-              <img
-                class="card-img-right"
-                src="../../../assets/field.png"
-                width="80px"
-                height="80px"
-              />
+<div class="d-none d-xl-block">
```

Comment:
Master data views should be constant. For example page /masterData/crop looks good:
![image](https://user-images.githubusercontent.com/30258800/95822185-b4065200-0d2b-11eb-9d1f-f7fe12fb0192.png)
Page /masterData/field should have the same template, instead:
![image](https://user-images.githubusercontent.com/30258800/95822270-dbf5b580-0d2b-11eb-83fa-ac7acde0568e.png)

Reply:
Resolved